### PR TITLE
Add Logging Improvements

### PR DESCRIPTION
**Pull request by ravmathu-amazon**
Adding Logging Improvements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,40 @@ To build the production version of this package, simply run `npm install && npm
 Import this into your package as is described in the GitHub examples.
 
 ## Customization
+### Logger Configuration
+The logger is provided by ChatJS package, you can configure it in this file: `src/utils/log.js`.
```
Comment (haomingli2020) [inline README.md:25]: nit: ChatJS -> amazon-connect-chatjs, can you also add a link here 
https://github.com/amazon-connect/amazon-connect-chatjs)[amazon-connect-chatjs](https://github.com/amazon-connect/amazon-connect-chatjs

Comment (haomingli2020) [inline README.md:25]: nit: ChatJS -> amazon-connect-chatjs, can you also add a link here 
https://github.com/amazon-connect/amazon-connect-chatjs)[amazon-connect-chatjs](https://github.com/amazon-connect/amazon-connect-chatjs

**Comment by haomingli2020** [comment]
Can you mention what's different from the internal CR in this PR?

**Comment by ravmathu-amazon** [comment]
> Can you mention what's different from the internal CR in this PR?

There are no differences in the PR from internal CR.

File: src/utils/log.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/* Add your own logger function here
+  var customizedLogger = {
+    debug: (data) => {// customize logger function here},
+    info: (data) => {// customize logger function here},
+    error: (data) => {// customize logger function here}
+  }
+  "connect" is a global object that exported from src/components/Chat/amazon-connect-chat.js
+  There are four levels available from connect.LogLevel - DEBUG, INFO, ERROR.
```
Comment (jiahaoyu6666) [inline src/utils/log.js:8]: nit: can you change "four" to "three"?

**Review by jiahaoyu6666 — APPROVED** [review]

**Review by haomingli2020 — APPROVED** [review]

File: src/utils/log.js (hunk 2 above)
Comment (ravmathu-amazon) [reply src/utils/log.js:8]: updated

**Pull request merged**